external/kafka-0-10/src/main/scala/org/apache/spark/streaming/kafka010/KafkaUtils.scala

```diff
@@ -177,3 +182,172 @@ object KafkaUtils extends Logging {
     }
   }
 }
+
+private[kafka010] class KafkaUtilsPythonHelper extends Logging {
+  import KafkaUtilsPythonHelper._
+
+  def createDirectStream(
+      jssc: JavaStreamingContext,
+      locationStrategy: LocationStrategy,
+      consumerStrategy: ConsumerStrategy[Array[Byte], Array[Byte]]
+    ): JavaDStream[Array[Byte]] = {
+    fixKafkaParams(consumerStrategy.executorKafkaParams)
```

Comment:
If we did a defensive copy on the map first, how do we set it back to `ConsumerStrategy`, I'm not pretty sure how to achieve this.